apps/src/aichat/views/tabs/tabs.module.scss

```diff
@@ -8,6 +8,7 @@
   padding-left: 5px;
   overflow-x: auto;
   overflow-y: hidden;
+  min-height: 30px;
```

Comment:
This semi-resolves an issues @fisher-alice noticed with tabs disappearing on smaller screens